cypress/integration-qe/e2e-enterClusterSpecs.spec.js

```diff
@@ -0,0 +1,98 @@
+import {
+  CLUSTER_NAME,
+  DNS_DOMAIN_NAME,
+  API_VIP,
+  INGRESS_VIP,
+  NUM_MASTERS,
+  NUM_WORKERS,
+  hostDetailSelector,
+  openCluster,
+  startClusterInstallation,
+  waitForClusterInstallation,
+} from './shared';
+
+describe('Enter cluster details', () => {
+  it('can open the cluster details', () => {
+    openCluster(CLUSTER_NAME);
+  });
+
+  it('wait for the hosts table to be populated', () => {
+    cy.get('table.hosts-table > tbody', { timeout: 10 * 60 * 1000 }).should(($els) => {
```

Comment:
nit: Can we put the timeout value to a constant and co-locate them? Please consider even introducing a new `constants.js` file for such things.

Reply:
Added constants.js and put the timeouts there